eater/models/partner.py

```diff
@@ -18,9 +18,53 @@ class Partner(models.Model):
         domain=[("customer", "=", True), ("eater", "=", "eater")],
     )
     parent_eater_id = fields.Many2one(
-        "res.partner", string="Parent Worker", readonly=True
+        "res.partner",
+        string="Parent Worker",
+        readonly=True,
+        domain=[("customer", "=", True), ("eater", "=", "worker_eater")],
```

Comment:
I think the domain condition on customer should be removed. There is some examples in our databases that the workers/eaters and the eaters are not customer (see on demain.test). So I think it will cause some worker/eater to not appear in the dropdown list.